Resources/public/js/views/services/ez-contentcreateviewservice.js

```diff
@@ -22,6 +22,20 @@ YUI.add('ez-contentcreateviewservice', function (Y) {
      * @extends eZ.ContentEditViewService
      */
     Y.eZ.ContentCreateViewService = Y.Base.create('contentCreateViewService', Y.eZ.ContentEditViewService, [], {
+        initializer: function () {
+            this.on('*:changeLanguage', this._changeLanguage);
+            this.after('*:languageCodeChange', function (e) {
+                var app = this.get('app'),
+                    activeView = app.get('activeView'),
+                    container = activeView.get('container'),
+                    languageContainer = container.one('.ez-content-current-language');
+
+                if (languageContainer) {
```

Comment:
you modifying the DOM in a view service that's bad because it's the responsibility of a view. Instead, you should add a method for that in the view and call it there.